### ISSUE TITLE
Windows 11 Menu

### ISSUE THREAD

**Issue by fadi9711** [Rectify11/Installer#426]
### Description

Not able to apply the Windows 11 default menu.

### Steps To Reproduce

Trying to apply the default Windows 11 menu from Rectify 11 Settings in the control panel, But it is not applying even other options, It is stuck on the enhanced fluent menu.

### Rectify11 Version

Version 3.1, 31 oct release

### Windows Version

Windows 11 23H2 22631.2861

### Logs

.

**Comment by MishaProductions** [comment]
This was fixed in latest actions build

**Comment by fadi9711** [comment]
Please, Can you tell me where Can I download this build?
Cause the last build on GitHub is on 31 of Oct.

Thank you

**Comment by MishaProductions** [comment]
This link should work: https://github.com/Rectify11/Installer/actions/runs/7439565885/artifacts/1152670138

**Comment by fadi9711** [comment]
Thank you so much it shows v3.2.60.0
can you please tell me How Can I get the update next time?

**Comment by fadi9711** [comment]
Sorry, I am asking a lot.
I updated to the version you sent and it is amazing and smoother, changing
the menu now is working fine, But there is one issue, Let's say I applied a
Nilesoft shell and then I turned my laptop off when I turned it on again or
if I made a restart it will revert to windows 11 default. Is this a
known issue?

Thank you

**Comment by MishaProductions** [comment]
I think it just takes some time to start

**Comment by fadi9711** [comment]
nope whenever you restart or if you restarted the explorer it will revert
to Windows 11 default
but there is a very important issue after installing this update it
affected WhatsApp if I tried to open the smiles menu it would freeze and if
I tried to open this menu also freeze and if I tried to share a file or
photo it would also freeze to make sure I uninstalled rectify the WhatsApp
back to normal now I re-installed rectify the WhatsApp problem back.
[image: image.png]

**Comment by VitalSkib** [comment]
> nope whenever you restart or if you restarted the explorer it will revert to Windows 11 default but there is a very important issue after installing this update it affected WhatsApp if I tried to open the smiles menu it would freeze and if I tried to open this menu also freeze and if I tried to share a file or photo it would also freeze to make sure I uninstalled rectify the WhatsApp back to normal now I re-installed rectify the WhatsApp problem back.

Yep, that's what I wanted to point out too. 
For whatever reason, after installing the Rectify11, the WhatsApp simply freezes if you use any additional functions. 
This applies to any of the Rectify11 latest versions from the Actions, but everything works fine with the Rectify11 v3.1 from the Releases section..

Windows 11 Pro Version	23H2
WhatsApp Latest from MS Store
Rectify11 Latest from Actions

**Comment by fadi9711** [comment]
Is there will be any fix for whatsapp issue? And also for the context menu
issue that whenever you restart it will back to default Windows 11?
And I found another problem that existed even with v3.1
I use Samsung Dex it was working normally but after installing rectify 11
now it has colored layer above it so you can see anything tried
uninstalling rectify but the problem still, tried re-installing dex also
didn't solve it, the only solution was clean windows instal, but when you
reinstall rectify the problem will back, So I preferred rectify 11 over Dex.
But I will really appreciate if you can look into it.



Thank you mr. DEV

**Comment by WinExperiments** [comment]
@fadi9711 Find the Samsung DeX process name, add it to MicaForEveryone's rules, and make sure "Extend Frame into Client Area" is disabled.

@VitalSkib For WhatsApp, try replacing C:\Windows\Resources\Themes\Rectified with the one from Version 3.1, reapply themes, and check if the issue persists.

**Comment by WinExperiments** [comment]
It would also be helpful to record the WhatsApp flyout animation, as that will help me locate what animation is used for it.

**Comment by VitalSkib** [comment]
> @VitalSkib For WhatsApp, try replacing C:\Windows\Resources\Themes\Rectified with the one from Version 3.1, reapply themes, and check if the issue persists.

Thanks, but I tried this before and it didn't help. 
I'll try this again..

**Comment by fadi9711** [comment]
Yeah sure I can help with the WhatsApp issue, but can you guide me how?
and Dex issue is fixed thank you so much

**Comment by fadi9711** [comment]
Sir please can you tell me where is this Actions section?
the only tap I see on GitHub is the releases section

**Comment by VitalSkib** [comment]
> Sir please can you tell me where is this Actions section? the only tap I see on GitHub is the releases section

https://github.com/Rectify11/Installer/actions

**Comment by VitalSkib** [comment]
> @VitalSkib For WhatsApp, try replacing C:\Windows\Resources\Themes\Rectified with the one from Version 3.1, reapply themes, and check if the issue persists.

Ha, it worked this time, thanks again. 😎👍
I wonder why this didn’t help me in the previous attempt and what’s in the new themes file that WhatsApp doesn’t like..

**Comment by WinExperiments** [comment]
@VitalSkib The animation map has changed between the visual styles' revisions. Maybe one of the new animations isn't supported by WhatsApp. Do you know which revision is the first to have the issue? (check version info of the Dark.msstyles file with Resource Hacker). Version 3.1 uses the 214th revision of the dark msstyles. Revision 216 and 220 are the ones publicly released after it. Also, record the flyout opening animation to help me finding out what the broken part in the animation map is.

**Comment by WinExperiments** [comment]
Can you record the animation under Version 3.1's themes? It will help me fix it.

**Comment by WinExperiments** [comment]
@VitalSkib Please respond as soon as possible.

**Comment by VitalSkib** [comment]
> Can you record the animation under Version 3.1's themes? It will help me fix it.

Animation of what exactly?

**Comment by VitalSkib** [comment]
> @VitalSkib Please respond as soon as possible.

If you meant WhatsApp, then here it is. 
If something else, then please explain more precisely.

https://github.com/Rectify11/Installer/assets/43848892/9a2fd10e-349c-466e-9bdb-e7abeba0b030

**Comment by WinExperiments** [comment]
@VitalSkib Thanks for the video. Can you also show how it looks on the default Windows theme?

**Comment by VitalSkib** [comment]
> @VitalSkib Thanks for the video. Can you also show how it looks on the default Windows theme?

Yes, sure, no problem. 
Although, to me it looks the same, there is no difference visually.
Here with the default Windows Dark Theme applied.

https://github.com/Rectify11/Installer/assets/43848892/d5bd80d4-56ea-4963-bd5d-589234a3ce3c

**Comment by WinExperiments** [comment]
@VitalSkib That looks the same… and I don't recall editing a Windows 11 animation in the visual styles. However, the newer installer may have installed the legacy version of the styles (designed for build 22000 and older). Can you please download Resource Hacker, open the .msstyles files with it, expand Version Info, and check if it says "legacy structure" or not?

**Comment by VitalSkib** [comment]
> @VitalSkib That looks the same… and I don't recall editing a Windows 11 animation in the visual styles. However, the newer installer may have installed the legacy version of the styles (designed for build 22000 and older). Can you please download Resource Hacker, open the .msstyles files with it, expand Version Info, and check if it says "legacy structure" or not?

Yes, but one problem, I have the latest version installed (from Actions), but I already replaced the Dark.msstyles file (the latest one from Release) and did not save the original one. 
![Resource Hacker - Dark msstyles](https://github.com/Rectify11/Installer/assets/43848892/e805ad57-8046-4afb-90b2-1b5bbbbe3082)

This is what the Resource Hacker shows for this file  (the latest one from Release).

**Comment by WinExperiments** [comment]
What about another file like Aero.msstyles?

**Comment by VitalSkib** [comment]
> What about another file like Aero.msstyles?


Sure, here are the rest of the files. 
And I don’t see this "legacy structure" entry here..

![Resource Hacker - DarkColorized msstyles](https://github.com/Rectify11/Installer/assets/43848892/8ab15046-a3f5-482d-a4ae-94f13513565d)
![Resource Hacker - Aero msstyles](https://github.com/Rectify11/Installer/assets/43848892/ee29bf27-b1b1-4c05-a255-a29cbeb02819)
![Resource Hacker - Black msstyles](https://github.com/Rectify11/Installer/assets/43848892/d6d39d9b-f500-42da-a302-f27b913f76ba)

**Comment by WinExperiments** [comment]
So it is really an animation I modified. Try this one (Dark216) and let me know.
[Dark.msstyles.zip](https://github.com/Rectify11/Installer/files/13929448/Dark.msstyles.zip)

**Comment by VitalSkib** [comment]
> So it is really an animation I modified. Try this one (Dark216) and let me know. [Dark.msstyles.zip](https://github.com/Rectify11/Installer/files/13929448/Dark.msstyles.zip)

Yep, this seems to fix the WhatsApp problem, but now it looks broken. 
Not Dark at all..

![Rectify11 Settings](https://github.com/Rectify11/Installer/assets/43848892/f29cd4b5-5fac-4eff-85e1-8ce626cfc13a)

**Comment by WinExperiments** [comment]
No… This is a broken mess consisting of the Windows default theme with some colors derived from Rectify11. Rename the file from Dark216.msstyles to Dark.msstyles and try again.

**Comment by VitalSkib** [comment]
> No… This is a broken mess consisting of the Windows default theme with some colors derived from Rectify11. Rename the file from Dark216.msstyles to Dark.msstyles and try again.

Of course I renamed this file before using it. 
But something is obviously wrong with it.. 
The dark theme just doesn't want to be applied. 
It either stays on the previous theme used or creates a mess like in my last screenshot.
So, for now I will use the file from the latest Release version.
PS I'm off for today..

**Comment by WinExperiments** [comment]
@VitalSkib @fadi9711 After extensive testing, I have found out what makes the WhatsApp menus not show: The custom taskbar animation. In more recent versions of Rectify11, the hidden taskbar's animation has been modified to fit more with Windows 11. For some reason, the WhatsApp menus use the animation that is only designed for the taskbar (by Microsoft), and WhatsApp does not like that animation modified. What can we expect from software written by Meta...

https://github.com/Rectify11/Installer/assets/91428431/21bbe43c-a25b-4728-89a5-c36017f60a58

**Comment by fadi9711** [comment]
@WinExperiments I am on the latest version of windows and I do not have this animation of the taskbar 
windows 11 23H2 (22631.3007)

**Comment by fadi9711** [comment]
@WinExperiments Hi 
I am on latest version of rectify 11 (3.2.60.0) and I do not have this animation, The theme applied is partial dark.
Also I have the context menu issue that still stuck on windows 11 default menu( I apply the Nilesoft menu and restart the explorer, it will be applied, but after restarting the pc it will go back to the default windows 11 menu).

**Comment by WinExperiments** [comment]
@fadi9711 Doesn't really matter since I'll remove the animation anyway.

**Comment by nasiftanjim** [comment]
anyone updates on this issue? I also have this whatsapp issue

**Comment by VitalSkib** [comment]
> anyone updates on this issue? I also have this whatsapp issue

Yes, it still hasn't been fixed, so just use version 3.1 from the Release section.

https://github.com/Rectify11/Installer/releases/tag/v-3.1-fixed2
